Makefile

```diff
@@ -1,4 +1,7 @@
 # Run jekyll in development mode
+
+LOCAL_DOCS_DIR := docs/v0.3.local
```

Comment:
does this "local docs" folder need to contain v0.3 explicitly? it would be nice not to hard code a version in there that we'll have to keep updating.

Reply:
@jbw976 easy fix, done